changelog.d/11865.removal

```diff
@@ -0,0 +1,4 @@
+Deprecate using `synctl` with the config option `synctl_cache_factor`.
+Print a warning if a user still uses this option. Instead suggest the user to: 
+ - Either set the environment variable 'SYNAPSE_CACHE_FACTOR'
+ - or set 'caches.global_factor' in the configs.
```

Comment:
I don't think lines 2-4 are needed-generally you don't need to add implementation details to a changelog. Perhaps something like "Deprecate using `synctl` with the config option `synctl_cache_factor` and print a warning if a user still uses this option" might suffice?

Reply:
OK